### PR TITLE
Contribution updates

### PR DESCRIPTION
**Pull request by simskij**
Adds a PR template asking the user to discuss changes either on slack or in an issue before getting started with their implementation. Adds the same text to the contribution guidelines as well.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  Thank you for your interest in contributing to the k6 project!
+  
+  Before you get started, we'd kindly like to ask you to read our:
+    - Contribution guidelines at https://github.com/loadimpact/k6/blob/master/CONTRIBUTING.md
+    - Code of Conduct at https://github.com/loadimpact/k6/blob/master/CODE_OF_CONDUCT.md
+    
+  Out of respect for your time, please start a discussion regarding any bigger contributions
+  either in the #contributors channel of the k6 slack or in the Github Issue before you get
+  started on the implementation.
+
```
Comment (na--) [inline .github/pull_request_template.md:15]: Maybe add a sentence at the end, something like
> If you've already done all of that, please go ahead and submit your PR and thank you for the contribution!

Doesn't sound quite right, but :man_shrugging:

**Review by na-- — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  (ﾉ◕ヮ◕)ﾉ*:・ﾟ✧
+  
+  Thank you for your interest in contributing to the k6 project!
+  
+  Before you get started, we'd kindly like to ask you to read our:
+    - Contribution guidelines at https://github.com/loadimpact/k6/blob/master/CONTRIBUTING.md
+    - Code of Conduct at https://github.com/loadimpact/k6/blob/master/CODE_OF_CONDUCT.md
+    
+  Out of respect for your time, please start a discussion regarding any bigger contributions
+  either in the #contributors channel of the k6 slack or in the Github Issue before you get
```
Comment (MStoykov) [inline .github/pull_request_template.md:13]: How about the community forum ?

**Review by MStoykov — APPROVED** [review]

File: .github/pull_request_template.md (hunk 2 above)
Comment (simskij) [reply .github/pull_request_template.md:13]: I can add that if it makes sense for you. 😅  I assumed (maybe incorrectly) that you, as core contributors, would prefer gh issues and slack.

File: .github/pull_request_template.md (hunk 1 above)
Comment (simskij) [reply .github/pull_request_template.md:15]: See proposed change 😄

File: .github/pull_request_template.md (hunk 2 above)
Comment (na--) [reply .github/pull_request_template.md:13]: for most things, github issue > community forum > slack - slack would only win over the community forum if near-real-time communication would be useful, for example to debug a tricky or urgent bug, or something like that

Comment (simskij) [reply .github/pull_request_template.md]: Done. Reapprove and i'll merge it 👍

**Review by na-- — APPROVED** [review]
LGTM, please squash when merging

**Pull request merged**